mgo.go

```diff
@@ -1,16 +1,20 @@
 // Copyright 2016 Canonical Ltd.
 // Licensed under the LGPLv3, see LICENCE file for details.
 
-package monitoring
+// Package mgomonitor provides prometheus metrics for statistics collected
+// by the https://godoc.org/gopkg.in/mgo.v2 package,
+// including items such as the number of current connections and
+// number of operations sent and received.
+package mgomonitor
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
 	"gopkg.in/mgo.v2"
 )
 
-// MgoStatsCollector is a prometheus.Collector that reports the values
+// Collector is a prometheus.Collector that reports the values
 // provided by mgo.GetStats.
-type MgoStatsCollector struct {
+type Collector struct {
```

Comment:
This assumes that we never want to support any of the other collectors. That's probably true, but I'm not certain we want to be stuck with that. How about just StatsCollector?

Reply:
In the unlikely event that we'll ever need two different MongoDB collectors, we can always choose a new name then. For the time being, I think that "Stats" as a qualifier is pretty meaningless - what would we be collecting from mgo other than statistics?